manticore/core/smtlib/expression.py

```diff
@@ -15,6 +15,14 @@ class ExpressionException(SmtlibError):
     pass
 
 
+class ExpressionEvalError(SmtlibError):
```

Comment:
I didn't really know how to name this exception. Something like `ExpressionConcretizeError` would be ambiguous IMO since `Concretize` is the exception that Manticore uses to fork and is totally unrelated to the error we want to address here.
Feel free to suggest a better name if you find one!

Reply:
This is probably fine for now 👍